scripts/generate_panelite_content.py

```diff
@@ -0,0 +1,24 @@
+"""
+Helper script to convert and copy example notebooks into JupyterLite build.
+"""
+import pathlib
+
+import nbformat
+
+PANEL_BASE = pathlib.Path(__file__).parent.parent
+EXAMPLES_DIR = PANEL_BASE / 'examples'
+DEPENDENCIES = ['panel', 'pyodide-http', 'altair', 'hvplot', 'matplotlib', 'plotly', 'pydeck', 'scikit-learn']
+
+nbs = list(EXAMPLES_DIR.glob('*/*/*.ipynb')) + list(EXAMPLES_DIR.glob('*/*.ipynb'))
+install = nbformat.v4.new_code_cell(source=f"import piplite\nawait piplite.install([{', '.join(DEPENDENCIES)}])")
```

Comment:
The dependencies needs to be wrapped in quotes here.

![image](https://user-images.githubusercontent.com/19758978/201998621-285c390a-7f84-4fac-937e-d456f0701560.png)

Reply:
Thanks.